addon/templates/components/freestyle/modals/ember-model.hbs

```diff
@@ -60,7 +60,7 @@
         buttonText="Open Modal"
       }}
       {{#freestyle-usage 'advanced-modal-ember-model' name='Advanced Modal - Ember Model'}}
-        {{#modals/ember-model
+        {{!-- {{#modals/ember-model
```

Comment:
Is there a reason you want to keep this in instead of removing it?

Reply:
No, meant to un-comment that.